x/oracle/client/cli/txRegisterOracle.go

```diff
@@ -16,9 +16,9 @@ import (
 
 func CmdRegisterOracle() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "register-oracle [unique ID] [node public key] [node public key remote report] [trusted block height] [trusted block hash] [endpoint] [oracle's commission rate]",
+		Use:   "register-oracle [unique-ID] [node-public-key] [node-public-key-remote-report] [trusted-block-height] [trusted-block-hash] [endpoint] [oracle-commission-rate] [oracle-commission-max-rate] [oracle-commission-max-change-rate]",
```

Comment:
I implemented this code to receive input as an argument, but it was better to change the flag.
Because it doesn't look good.
wdyt?

Reply:
But it doesn't matter. :)

Reply:
Thanks, I changed it using flag ([ab40af0](https://github.com/medibloc/panacea-core/pull/540/commits/ab40af0819b45dee477db3cb1317e287c0a0efdd)).